PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -163,9 +163,44 @@ def miniAOD_customizeCommon(process):
         cms.InputTag("reducedEgamma","reducedEERecHits")
     process.electronIDValueMapProducer.esReducedRecHitCollection = \
         cms.InputTag("reducedEgamma","reducedESRecHits")
+    process.electronMVAValueMapProducer.src = \
+        cms.InputTag("reducedEgamma","reducedGedGsfElectrons")
+    process.electronRegressionValueMapProducer.src = \
+        cms.InputTag("reducedEgamma","reducedGedGsfElectrons")
+    process.electronRegressionValueMapProducer.ebReducedRecHitCollection = \
```

Comment:
The rechits collections you point at don't matter so much.
So this is OK, but it is indeed likely causing conflicts.